### PR TITLE
Refactor: refine the amendments add and edit page.

### PR DESCRIPTION
**Pull request by stu01509**
#### Short description of what this resolves:
Refine the amendments add and edit page.

Add the bootstrap grid system.
Add the bootstrap table and form style.
Remove outdated code and style.

image

image

### CODE REVIEW

File: interface/patient_file/summary/add_edit_amendments.php [hunk 1]
```diff
@@ -116,9 +116,6 @@
 <?php Header::setupHeader('datetime-picker'); ?>
 
 <style>
-.highlight {
-  color: var(--success);
-}
 tr.selected {
   background-color: var(--white);
 }
```
Comment (stu01509) [inline interface/patient_file/summary/add_edit_amendments.php:121]: Hi @tywrenn 

I guess this CSS style can be replaced with `table-hover`, What do you think?

Comment (tywrenn) [reply interface/patient_file/summary/add_edit_amendments.php:121]: Yep that can be replaced with `table-hover`

**Comment by tywrenn** [comment]
@stu01509 Looks like this is 34 commits behind master and needs rebasing before I review it

**Comment by tywrenn** [comment]
@stu01509 Something doesn't look right here still (i see code from another pull in here) -- might wanna close and reopen

**Comment by stu01509** [comment]
Close this PR, and create another PR in #3753

**Pull request closed without merging**